Marlin/Configuration.h

```diff
@@ -1284,12 +1284,13 @@
 //#define MESH_BED_LEVELING
 
 /**
- * Normally G28 leaves leveling disabled on completion. Enable one of
- * these options to restore the prior leveling state or to always enable
- * leveling immediately after G28.
+ * Normally G28 leaves leveling disabled on completion.
+ * Enable one of these options to change behaviour:
+ * ACTIVATE_LVL   = force leveling activation
+ * DEACTIVATE_LVL = keep leveling deactivated
+ * RESTORE_LVL    = restore previous leveling state
  */
-//#define RESTORE_LEVELING_AFTER_G28
-//#define ENABLE_LEVELING_AFTER_G28
+//#define LEVELING_AFTER_G28
```

Comment:
I think some opinion is needed here to decide which would be the better option:

- Add one of the three option as default e.g. `//#define LEVELING_AFTER_G28 RESTORE_LVL` or
- Leave `//#define LEVELING_AFTER_G28` as is.
If it is left empty, users might blindly enable it, not seeing that it needs one of the three options in order to not get sanitycheck error (https://github.com/MarlinFirmware/Marlin/pull/20479#issuecomment-745424808), and most likely will open a new issue without reading their error message.

Reply:
Now sanity check should signal such case

Reply:
Yes, on an option error the sanitycheck error message gives enough information to solve the problem, but we still get many issues open because users don't read the error messages.

Reply:
As `LEVELING_AFTER_G34` and `G35` both are `RESTORE_LVL` as default, then shouldn't `G28` be that too?

Reply:
No, they are set to keep backward compatibility

Reply:
I still think that it might be better to have a default disabled value for `LEVELING_AFTER_G28`, instead of leave it blank, e.g. `RESTORE_LVL`, as that option was the only one for a long time (if wasn't set false), but now this depends on the reviewers.

Reply:
> but now this depends on the reviewers.

exactly